.github/workflows/stale-closer.yaml

```diff
@@ -13,13 +13,13 @@ jobs:
     steps:
       - uses: actions/stale@v5
         with:
-          days-before-issue-stale: 30
+          days-before-issue-stale: 60
           days-before-issue-close: 30
           stale-issue-label: "stale"
-          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity."
-          stale-pr-message: "This PR is stale because it has been open for 30 days with no activity."
+          stale-issue-message: "This issue is stale because it has been open for 60 days with no activity."
+          stale-pr-message: "This PR is stale because it has been open for 60 days with no activity."
           close-issue-message: "This issue was closed because it has been inactive for 30 days since being marked as stale."
           close-pr-message: "This PR was closed because it has been inactive for 30 days since being marked as stale."
-          days-before-pr-stale: 30
+          days-before-pr-stale: 60
```

Comment:
Will this remove the stale label from PRs and issues between 30 days and 60 days? Or should we review and manually update?

Reply:
I don't think it will do it auto but not sure